authors/tests.py

```diff
@@ -14,24 +14,28 @@ def setup_single_user_and_author(self):
         self.user = User.objects.create_user('test_username', 'test_email', 'test_pass')
         self.author = Author.objects.create(user=self.user, display_name=self.user.username)
     def setUp(self):
-        self.setup_single_user_and_author()
+        pass
 
     def test_get_author_list(self):
+        self.setup_single_user_and_author()
         res = client.get('/authors/', format='json')
         content = json.loads(res.content)
 
-        # content should look like [{'id': 'adfsadfasdfasdf', 'display_name': 'test_username', 'url': '', 'host': '', 'user': 1, 'friends': []}]
+        # content should look like [{'id': 'adfsadfasdfasdf', 'displayName': 'test_username', 'url': '', 'host': '', 'user': 1, 'friends': []}]
         assert len(content) == 1
-        assert content[0]['display_name'] == 'test_username'
-        assert content[0]['user'] == self.user.pk
+
+        # API fields as per spec, not model fields.
+        assert content[0]['displayName'] == 'test_username'
+        assert content[0]['id'] == str(self.author.id)
         assert res.status_code == 200
         
     def test_get_author_detail(self):
+        self.setup_single_user_and_author()
         res = client.get(f'/author/{self.author.id}/', format='json')
         content = json.loads(res.content)
 
-        # content should look like {'id': 'adfsadfasdfasdf', 'display_name': 'test_username', 'url': '', 'host': '', 'user': 1, 'friends': []}
-        assert content['display_name'] == 'test_username'
-        assert content['user'] == self.user.pk
+        # content should look like {'id': 'adfsadfasdfasdf', 'displayName': 'test_username', 'url': '', 'host': '', 'user': 1, 'friends': []}
+        assert content['displayName'] == 'test_username'
```

Comment:
```suggestion
        assert content['display_name'] == 'test_username'
```

Reply:
we are testing API endpoint. According to the spec it probably should stick to 
![image](https://user-images.githubusercontent.com/25297856/135188585-d44d95f0-ee40-426d-aea7-06098f016b6a.png)

`displayName`

Reply:
oh the spec uses camelCase, that can be a bit confusing.. but we'll stick with the spec for now